.config/nextest.toml

```diff
@@ -0,0 +1,9 @@
+[profile.default]
+retries = 1
+
+[profile.ci]
```

Comment:
Can we also set the `no-fail-fast` option here? Otherwise I think the tests will terminate once a failure is encountered.